dev/devicelab/lib/framework/devices.dart

```diff
@@ -852,16 +850,12 @@ class IosDevice extends Device {
         'DYLD_LIBRARY_PATH': dyldLibraryPath,
       },
     );
-    _loggingProcess!.stdout
-      .transform<String>(const Utf8Decoder(allowMalformed: true))
-      .listen((String line) {
-        sink.write(line);
-      });
-    _loggingProcess!.stderr
-      .transform<String>(const Utf8Decoder(allowMalformed: true))
-      .listen((String line) {
-        sink.write(line);
-      });
+    _loggingProcess!.stdout.transform<String>(const Utf8Decoder(allowMalformed: true)).listen((String line) {
```

Comment:
ditto

Reply:
Reverted back to previous formatting to simplify the code review